src/coreclr/inc/corinfo.h

```diff
@@ -1700,27 +1700,23 @@ struct CORINFO_EE_INFO
 
 enum CorInfoContinuationFlags
 {
-    // Whether or not the continuation expects the result to be boxed and
-    // placed in the GCData array at index 0. Not set if the callee is void.
-    CORINFO_CONTINUATION_RESULT_IN_GCDATA = 1,
+    CORINFO_CONTINUATION_HAS_OSR_ILOFFSET = 1,
     // If this bit is set the continuation resumes inside a try block and thus
     // if an exception is being propagated, needs to be resumed. The exception
     // should be placed at index 0 or 1 depending on whether the continuation
     // also expects a result.
     CORINFO_CONTINUATION_NEEDS_EXCEPTION = 2,
```

Comment:
Naming nit: All other enum members are `HAS_...,` this one is `NEEDS_... `. Should this one be called HAS_EXCEPTION?